src/WebApp/flask/aztk_cluster.py

```diff
@@ -74,12 +74,16 @@ def createCluster(self):
             jars=jars
         )
 
+        clusterDetails = self.table_service.get_entity('cluster', 'predictivemaintenance', 'predictivemaintenance')
```

Comment:
what if no cluster has been provisioned?

Reply:
The status will be NotCreated so it will show the create cluster page.